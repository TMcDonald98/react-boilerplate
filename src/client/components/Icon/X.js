
import * as React from 'react';

export const X = () => (
  <svg className='x-icon' width='24' height='24' viewBox='0 0 24 24'>
    <g transform='translate(1 1)' stroke='#F35454' fill='none' fillRule='evenodd'>
      <circle cx='11' cy='11' r='11'/>
      <path d='M5.5 5.5l11 11M5.5 16.5l11-11' strokeLinecap='round' />
    </g>
  </svg>
);

export default X;
