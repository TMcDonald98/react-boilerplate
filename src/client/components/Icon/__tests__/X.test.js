import * as React from 'react';
import { X } from '../X';

describe('components/Icon/X', () => {
  describe('render', () => {
    it('should render without error', () => {
      const result = shallow(<X />);

      expect(result).toMatchSnapshot('X');
    });
  });
});
