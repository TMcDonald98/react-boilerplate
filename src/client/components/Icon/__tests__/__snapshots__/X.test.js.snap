// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Icon/X render should render without error: X 1`] = `
<svg
  className="x-icon"
  height="24"
  viewBox="0 0 24 24"
  width="24"
>
  <g
    fill="none"
    fillRule="evenodd"
    stroke="#F35454"
    transform="translate(1 1)"
  >
    <circle
      cx="11"
      cy="11"
      r="11"
    />
    <path
      d="M5.5 5.5l11 11M5.5 16.5l11-11"
      strokeLinecap="round"
    />
  </g>
</svg>
`;
